src/main/java/mate/jdbc/service/DriverServiceImpl.java

```diff
@@ -0,0 +1,40 @@
+package mate.jdbc.service;
+
+import java.util.List;
+import mate.jdbc.dao.DriverDao;
+import mate.jdbc.dao.DriverDaoImpl;
+import mate.jdbc.lib.Inject;
+import mate.jdbc.lib.Service;
+import mate.jdbc.model.Driver;
+
+@Service
+public class DriverServiceImpl implements DriverService {
+    @Inject
+    private static final DriverDao driverDao = new DriverDaoImpl();
```

Comment:
it should be non-static field. Also, you have annotation `@inject`, you don't need to hardcode implementation 
```suggestion
    private DriverDao driverDao;
```